package.json

```diff
@@ -22,8 +22,8 @@
     "test:browser": "npx jest --runInBand --verbose --config test/browser/jest.config.js --forceExit",
     "test:browser:ci": "npm run install:puppeteer && npm run test:browser",
     "install:puppeteer": "npm install --save-dev jest-puppeteer puppeteer @types/puppeteer",
-    "prepare": "node ./scripts/tsc.js && node ./scripts/build.js",
```

Comment:
prepare runs both during packaging and when user installs on the system. we probably dont need it for packing, but wouldnt we want to have it for when user install and binaries are not available for their system?

Reply:
User will install the package from [npm install](https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-install), which will run the install script and build the native addon if not existed. 

for npm publish and npm pack, I don't believe our user should ever use those.

But, after double check the documentation, maybe we want to support https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-cache-add?

Reply:
in the failing release pipeline, why isn't install.js just skipping the cmake build? Doesn't it just skip building if the binary is already there?